kubernetes/values.yaml

```diff
@@ -0,0 +1,20 @@
+# Default values for torchserve helm chart.
+
+torchserve_image: pytorch/torchserve:latest-gpu
+
+namespace: torchserve
+
+torchserve:
+  management_port: 8081
+  inference_port: 8080
+  pvd_mount: /home/model-server/shared/
+  n_gpu: 1
+  n_cpu: 1
+  memory_limit: 4Gi
+  memory_request: 1Gi
+
+deployment:
+  replicas: 1
```

Comment:
@dhanainme Mention that the replicas value needs to be <= the total number of worker nodes in the Kubernetes cluster

Reply:
Done